src/helper/engine_settings.hpp

```diff
@@ -58,14 +73,18 @@ struct engine_settings {
                   << ", trace_rate_limit=" << c.trace_rate_limit
                   << ", obfuscator_key_regex=" << c.obfuscator_key_regex
                   << ", obfuscator_value_regex=" << c.obfuscator_value_regex
-                  << "}";
+                  << ", schema_extraction.enabled="
+                  << c.schema_extraction.enabled
+                  << ", schema_extraction.sample_rate=" << std::fixed
```

Comment:
What's `std::fixed` for?

Reply:
https://cplusplus.com/reference/ios/fixed/